makemkv/output_codes.py

```diff
@@ -11,19 +11,31 @@
     2: "name",
     3: "langcode",
     4: "language",
-    5: "codec",
+    5: "codec_id",
+    6: "codec_short",
+    7: "codec_long",
     8: "chapter_count",
     9: "length",
     10: "size_human",
     11: "size",
     13: "bitrate",
     15: "video_angle",
+    16: "source_filename",
     17: "samplerate",
     19: "dimensions",
     20: "aspect_ratio",
     21: "framerate",
+    22: "stream_flags",
+    23: "date_time",
+    24: "original_title_id",
```

Comment:
I guess this should be referenced in `makemkv.types.Title`.

Reply:
i removed date_time and original_title_id